tests/torch/fx/test_models.py

```diff
@@ -157,32 +166,46 @@ def test_model(test_case: ModelCase):
             (250, 250),
             (149, 149),
         ],
+        [True, False, True, True],
     ),
     (
         ModelCase(partial(ShortTransformer, 5, 10), "synthetic_transformer", [5]),
         {"model_type": nncf.ModelType.TRANSFORMER},
         [(4, 4), (2, 2)],
+        [True],
```

Comment:
Can we build dynamic shapes directly here, without bool -> dim conversion?
```suggestion
        [Dim.Auto],
```

Reply:
Yes, that is possible. I used bool since it was simpler type and to avoid repetition. I will change it as suggested

Reply:
Done